### PR TITLE
refactor(gui): revert drive/image selection step order swap

### PR DESCRIPTION
**Pull request by Shou**
We revert the drive and image selection step order swap back from drive
selection first and image second, to image first and drive second.

See: https://github.com/resin-io/etcher/pull/1049
Changelog-Entry: Revert the drive/image selection step order swap.

### CODE REVIEW

File: lib/gui/css/main.css [hunk 1]
```diff
@@ -6402,30 +6402,30 @@ body {
 .component-drive-selector-body .list-group-item-text {
   word-break: break-all; }
 
-.modal-drive-selector-modal .list-group {
+.component-drive-selector-body .list-group {
```
Comment (jviotti) [inline lib/gui/css/main.css]: Where are these CSS changes coming from? Looks like this is accidentally including unrelated changes.

**Comment by jviotti** [comment]
screenshot 2017-02-08 10 09 16

I think the drive step should be disabled by default in that case ^

File: lib/gui/css/main.css (hunk 1 above)
Comment (Shou) [inline lib/gui/css/main.css]: Yeah, that's strange.

Comment (Shou) [inline lib/gui/css/main.css]: Maybe someone forgot to `npm run sass` in a previous commit? Running it again changes nothing, and it's based on `master` as far as I'm aware...

Comment (jviotti) [inline lib/gui/css/main.css]: OK, cool. Can we send another PR to "re-build CSS" and omit these changes from here?

**Comment by Shou** [comment]
I'll rebase this on master once #1088 is merged.

File: lib/gui/css/main.css (hunk 1 above)
Comment (lurch) [inline lib/gui/css/main.css]: It looks like this might be related to #1073 ? But that PR _does_ also modify `lib/gui/css/main.css`, which is weird.

Observe:
```bash
andrew@shyknee:~/github/resin-io/etcher-clean$ git checkout 05fdd5c
Previous HEAD position was 70c7ddc... fix(GUI): don't auto select system drives in unsafe mode
HEAD is now at 05fdd5c... refactor(GUI): unify modal body styles (#1073)
andrew@shyknee:~/github/resin-io/etcher-clean$ git status
HEAD detached at 05fdd5c
nothing to commit, working directory clean
andrew@shyknee:~/github/resin-io/etcher-clean$ npm run sass

> etcher@1.0.0-beta.18 sass /home/andrew/github/resin-io/etcher-clean
> node-sass ./lib/gui/scss/main.scss > ./lib/gui/css/main.css

andrew@shyknee:~/github/resin-io/etcher-clean$ git status
HEAD detached at 05fdd5c
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   lib/gui/css/main.css

no changes added to commit (use "git add" and/or "git commit -a")
```

Is it possible that @jviotti somehow has a different version of `sass` installed, and that's why #1073 didn't "properly" update the CSS? :-/

Comment (jviotti) [inline lib/gui/css/main.css]: I have no clue. Maybe I forgot to re-run `sass` after some more changes?

Comment (lurch) [inline lib/gui/css/main.css]: Haha, another bug in @jviotti ? :laughing:

**Comment by Shou** [comment]
Here we go. 

image

**Comment by lurch** [comment]
Has the tooltip icon (that gave you the image's full path) suddenly disappeared?
(or have I not been keeping up, and that was removed in a different PR?)

File: lib/gui/css/main.css (hunk 1 above)
Comment (jviotti) [inline lib/gui/css/main.css]: lol

**Comment by jviotti** [comment]
@lurch Yeah, it was removed, but clicking on the image name / size still reveals it.
